### DISCUSSION TITLE
Switch from "for you" tab to "following" tab as default?

### DISCUSSION THREAD

**Discussion by stellayurigasaki** [AprilSylph/XKit-Rewritten#1258] [Suggestion Box]
Has there been consideration about a fix for the "for you" tab Tumblr implemented as the default display for blogs created after May 8th? A lot of tumblr users don't like the fact that the "for you" tab is being shoved down their throats but I've yet to see anyone create a code or a script to undo the change.

**Comment by AprilSylph** [comment]
Duplicate of #1158. Thanks for reminding me that I need to review the solution to that issue.
